.github/workflows/tag.yml

```diff
@@ -0,0 +1,104 @@
+# NB: release notes inspired by https://blogs.sap.com/2018/06/22/generating-release-notes-from-git-commit-messages-using-basic-shell-commands-gitgrep/
+name: Tag Creator
+on:
+  issues:
+    types: [opened]
```

Comment:
I understand why this is `opened` and not `closed` but it feels like it would be nice to have some way to have group consensus in the public record before this gets kicked off.